### PR TITLE
newsletter 8/7/18

### PR DESCRIPTION
**Pull request by mikemcphe**
uploading the newsletter for 8/7/18. can't push directly because master is a protected branch

### CODE REVIEW

File: content/newsletters/2018-08-07-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
++++
+
+title = "2018-08-07 Newsletter"
+template = "post.html"
+date = 2018-08-07
+slug = "newsletter"
+tags = [ "systems", "website" ]
```
Comment (remexre) [inline content/newsletters/2018-08-07-newsletter.md]: why these tags?

**Review by remexre — CHANGES_REQUESTED** [review]

File: content/newsletters/2018-08-07-newsletter.md (hunk 1 above)
Comment (mzhang28) [reply content/newsletters/2018-08-07-newsletter.md]: d

Comment (mikemcphe) [reply content/newsletters/2018-08-07-newsletter.md]: those were the tags in the example post

Comment (remexre) [reply content/newsletters/2018-08-07-newsletter.md]: change to `tags = []` or choose different tags

**Review by remexre — APPROVED** [review]

**Pull request merged**